framework/auth/core.py

```diff
@@ -147,10 +147,11 @@ def get_user(email=None, password=None, verification_key=None):
 class Auth(object):
 
     def __init__(self, user=None, api_node=None,
-                 private_key=None):
+                 private_key=None, private_link=None):
```

Comment:
And with a533833fcdba76954c1cab853d1caf920c70db22 I believe we no longer need the `private_link` kwarg here.